src/index.js

```diff
@@ -1,14 +1,20 @@
-require('./app')
-require('./api')
-require('./babbage')
-require('./chart')
-require('./crosstab')
-require('./facts')
-require('./pager')
-require('./panel')
-require('./sankey')
-require('./treemap')
-require('./util')
-require('./workspace')
+//require('./app')
```

Comment:
@borysyuk please remove *all* code we are no longer using. Do a new PR for that.